api/helpers.go

```diff
@@ -29,6 +30,14 @@ func getToken(ctx context.Context) *jwt.Token {
 	return obj.(*jwt.Token)
 }
 
+func getConfig(ctx context.Context) *conf.Configuration {
```

Comment:
instead of doing this as a helper, you could create a Context that intentionally shadows the `context.Context` and has methods `WithConfig(conf *conf.Configuration)` and a `GetConfig(conf *conf.Configration)` so you can directly set/get the config. 

https://github.com/netlify/netlifyctl/blob/master/context/context.go